assets/sendmail_template.txt

```diff
@@ -9,12 +9,12 @@ Content-Type: text/html; charset=utf-8
 $email_html
 
 --nfcoremimeboundary
-Content-Type: image/png;name="nfcore-rnaseq_logo.png"
+Content-Type: image/png;name="nf-core-rnaseq_logo.png"
```

Comment:
I guess that you can delete `nfcore-rnaseq_logo.png` now? Or maybe rename it to replace the automatically generated one with this (illustrator probably does a nicer job of resizing images than PHP).

Reply:
I just renamed to the new format too.

Reply:
I have to say I do prefer the older logos 👀

Reply:
Individual care in illustrator >> script automation.. :( Not sure I can do any better than what we have now sadly.

Hmm, except if I create an image at the final height instead of making a big one and resizing. Then that could potentially look better..